src/main/webapp/WEB-INF/tags/instructor/remindParticularStudentsModal.tag

```diff
@@ -16,6 +16,16 @@
             Remind Particular Students
             <small>(Select the student(s) you want to remind)</small>
           </h4>
+          <div>
+            <input id="remind_all" type="checkbox" value="">
```

Comment:
Please use camelCase for `id` for consistency with the rest of the code base.

Reply:
Also, if `value` is empty is it necessary here?

Reply:
i see. sorry for the overlook, removed value as well.